### PR TITLE
DOCSP-41968: Update documents

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-php-library/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-41968
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/php-library/DOCSP-41968-update/write/update/

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/write/update.txt [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+  perform, and the fields and values to change. For a list of update operators
+  and their usage, see the :manual:`Field Update Operators guide
+  </reference/operator/update-field/>` in the {+mdb-server+} manual.
+
+Update One Document
+~~~~~~~~~~~~~~~~~~~
+
+The following example uses the ``updateOne()`` method to update the ``name``
+value of a document in the ``restaurants`` collection from ``'Bagels N Buns'``
+to ``'2 Bagels 2 Buns'``:
```
Comment (mcmorisi) [inline source/write/update.txt:72]: I chuckled

File: source/write/update.txt [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+- **Update** document: Specifies the **update operator**, or the kind of update to
+  perform, and the fields and values to change. For a list of update operators
+  and their usage, see the :manual:`Field Update Operators guide
+  </reference/operator/update-field/>` in the {+mdb-server+} manual.
+
+Update One Document
+~~~~~~~~~~~~~~~~~~~
+
+The following example uses the ``updateOne()`` method to update the ``name``
```
Comment (mcmorisi) [inline source/write/update.txt:70]: Q: Do we need to refer to this with the full `MongoDB\Collection::updateOne()` name every time? There's a few other occurrences in this PR where we don't use the long method name as well.

File: source/write/update.txt [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+.. literalinclude:: /includes/write/update.php
+   :start-after: start-update-one
+   :end-before: end-update-one
+   :language: php
+   :dedent:
+
+Update Many Documents
+~~~~~~~~~~~~~~~~~~~~~
+
+The following example uses the ``updateMany()`` method to update all documents
```
Comment (mcmorisi) [inline source/write/update.txt:83]: Same question re: long/short name

File: source/write/update.txt [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+.. literalinclude:: /includes/write/update.php
+   :start-after: start-update-many
+   :end-before: end-update-many
+   :language: php
+   :dedent:
+
+Customize the Update Operation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can modify the behavior of the ``updateOne()`` and ``updateMany()`` methods by
```
Comment (mcmorisi) [inline source/write/update.txt:96]: Same question re: long/short name

File: source/write/update.txt [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+         fields. For more information, see the :manual:`let statement
+         </reference/command/update/#std-label-update-let-syntax>` in the
+         {+mdb-server+} manual.
+
+   * - ``comment``
+     - | A comment to attach to the operation. For more information, see the :manual:`insert command
+         fields </reference/command/insert/#command-fields>` guide in the
+         {+mdb-server+} manual.
+
+The following example uses the ``updateMany()`` method to find all documents that
```
Comment (mcmorisi) [inline source/write/update.txt:153]: Same question re: long/short name

File: source/write/update.txt [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+     - | Returns a boolean indicating whether the write operation was acknowledged.
+
+   * - ``getUpsertedCount()``
+     - | Returns the number of document that were upserted into the database.
+
+   * - ``getUpsertedId()``
+     - | Returns the ID of the document that was upserted in the database, if the driver
+         performed an upsert.
+        
+The following example uses the ``updateMany()`` method to update the ``name`` field
```
Comment (mcmorisi) [inline source/write/update.txt:198]: Same question re: long/short name

**Review by mcmorisi — APPROVED** [review]
LGTM with just one question

File: source/write/update.txt (hunk 2 above)
Comment (norareidy) [reply source/write/update.txt:70]: I've been using the full name for first mentions in each top-level section, then the shortened name for future mentions in that section and in its subsections.

Comment (mcmorisi) [reply source/write/update.txt:70]: Makes sense to me and looks like this page adheres to that rule.

**Review by mcmorisi — APPROVED** [review]
LGTM!

**Review by alcaeus — APPROVED** [review]

**Pull request merged**